src/api/app/views/webui2/webui/request/_decision_tab.html.haml

```diff
@@ -1,13 +1,20 @@
 = form_tag({ action: 'changerequest' }, id: 'request_handle_form') do
   = hidden_field_tag(:number, request_number)
   %p
-    = text_area_tag(:reason, nil, placeholder: 'Please add a comment', row: 4, class: 'w-100 form-control')
-  - if state.in?(['new', 'review']) && maintainer_option && !action[:creator_is_target_maintainer]
-    .custom-control.custom-checkbox
-      = check_box_tag('add_submitter_as_maintainer_0', "#{action[:tprj]}_#_#{action[:tpkg]}", false, class: 'custom-control-input')
-      %label.custom-control-label{ for: 'add_submitter_as_maintainer_0' }
-        Add #{link_to(request_creator, user_show_path(request_creator))} as maintainer for
-        #{project_or_package_link(project: action[:tprj], package: action[:tpkg], short: true)}
+    = text_area_tag(:reason, nil, placeholder: 'Please add a comment', rows: 4, class: 'w-100 form-control')
+  - if single_action_request
+    - if state.in?(['new', 'review']) && !action[:creator_is_target_maintainer]
+      .custom-control.custom-checkbox
+        = check_box_tag('add_submitter_as_maintainer_0', "#{action[:tprj]}_#_#{action[:tpkg]}", false, class: 'custom-control-input')
```

Comment:
This should be `add_submitter_as_maintainer_#{index}` :thinking:

Reply:
This option only works for requests with single actions. That's also the case in the old UI. I just kept the format of the parameter this way so I don't have to change the old code.